owncloud/ios-app
# Correct handling for duplicated file names

## Description
Fixed action `Duplicate` to get the correct file name for a duplicated file or folder.
Now the `macOS` `Finder` logic is used.

## Related Issue
#320 

## Motivation and Context
App showed error message, if file already exists.

## How Has This Been Tested?
1. Open file list
2. Select one ore more files
3. Files should be duplicated with new file name
4. `Paris.jpg` > `Paris copy.jpg`
5. `Paris copy.jpg` > `Paris copy 1.jpg`
6. `Paris.jpg` > `Paris copy 2.jpg`
7. `Paris copy 1 .jpg` > `Paris copy 3.jpg`

Use the same handling as `macOS` `Finder`.

## Types of changes


## Checklist:



Test plan: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Duplicate%20II.md
